ios/.gitignore

```diff
@@ -0,0 +1,37 @@
+.idea/
```

Comment:
No, I didn't add them; that's just what was generated. I've added those directories to gitignore!